integration_tests/e2e/induction/updateCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -0,0 +1,94 @@
+
+    // When
+    // Change Hoping To Work On Release
+    Page.verifyOnPage(CheckYourAnswersPage)
+    checkYourAnswersPage
+      .clickHopingToWorkOnReleaseChangeLink()
+      .hasBackLinkTo(`/prisoners/${prisonNumber}/induction/check-your-answers`)
+      .selectHopingWorkOnRelease(HopingToGetWorkValue.NOT_SURE)
+      .submitPage()
+
+    // Change Reasons For Not Wanting To Work
+    Page.verifyOnPage(CheckYourAnswersPage)
+    checkYourAnswersPage
+      .clickReasonsForNotWantingToWorkChangeLink()
+      .hasBackLinkTo(`/prisoners/${prisonNumber}/induction/check-your-answers`)
+      .deSelectReasonNotToGetWork(ReasonNotToGetWorkValue.HEALTH)
+      .chooseReasonNotToGetWork(ReasonNotToGetWorkValue.RETIRED)
+      .chooseReasonNotToGetWork(ReasonNotToGetWorkValue.NO_REASON)
+      .submitPage()
+
+    // Change Other Training
+    Page.verifyOnPage(CheckYourAnswersPage)
+    checkYourAnswersPage
+      .clickAdditionalTrainingChangeLink()
+      .hasBackLinkTo(`/prisoners/${prisonNumber}/induction/check-your-answers`)
+      .deSelectAdditionalTraining(AdditionalTrainingValue.FULL_UK_DRIVING_LICENCE)
+      .deSelectAdditionalTraining(AdditionalTrainingValue.HGV_LICENCE)
+      .deSelectAdditionalTraining(AdditionalTrainingValue.OTHER)
+      .chooseAdditionalTraining(AdditionalTrainingValue.MANUAL_HANDLING)
+      .chooseAdditionalTraining(AdditionalTrainingValue.CSCS_CARD)
+      .submitPage()
+
+    // Change In Prison Work Interests
+    Page.verifyOnPage(CheckYourAnswersPage)
+    checkYourAnswersPage
+      .clickInPrisonWorkInterestsChangeLink()
+      .hasBackLinkTo(`/prisoners/${prisonNumber}/induction/check-your-answers`)
+      .deSelectWorkType(InPrisonWorkValue.CLEANING_AND_HYGIENE)
+      .chooseWorkType(InPrisonWorkValue.PRISON_LAUNDRY)
+      .chooseWorkType(InPrisonWorkValue.PRISON_LIBRARY)
+      .submitPage()
+
+    // Change In Prison Training Interests
+    Page.verifyOnPage(CheckYourAnswersPage)
+    checkYourAnswersPage
+      .clickInPrisonTrainingInterestsChangeLink()
+      .hasBackLinkTo(`/prisoners/${prisonNumber}/induction/check-your-answers`)
+      .deSelectInPrisonTraining(InPrisonTrainingValue.BARBERING_AND_HAIRDRESSING)
+      .chooseInPrisonTraining(InPrisonTrainingValue.CATERING)
+      .chooseInPrisonTraining(InPrisonTrainingValue.NUMERACY_SKILLS)
+      .submitPage()
+
+    // Then
+    Page.verifyOnPage(CheckYourAnswersPage) //
+      .hasHopingToWorkOnRelease(HopingToGetWorkValue.NOT_SURE)
+      .hasReasonsForNotWantingToWork([ReasonNotToGetWorkValue.RETIRED, ReasonNotToGetWorkValue.NO_REASON])
+      .hasAdditionalTraining([AdditionalTrainingValue.MANUAL_HANDLING, AdditionalTrainingValue.CSCS_CARD])
+      .hasInPrisonWorkInterests([InPrisonWorkValue.PRISON_LAUNDRY, InPrisonWorkValue.PRISON_LIBRARY])
+      .hasInPrisonTrainingInterests([InPrisonTrainingValue.CATERING, InPrisonTrainingValue.NUMERACY_SKILLS])
+  })
+
+  // TODO - RR-736 Implement this test
```

Comment:
This is the Long Question Set test. Hopefully there is enough here to be able to get the gist of what needs to happen in this test 🤞